dashboard/client/package.json

```diff
@@ -29,7 +31,7 @@
       "pathRewrite": {
         "^/api": "/"
       },
-      "target": "http://127.0.0.1:8080/function"
+      "target": "http://localhost:7777/function/"
```

Comment:
on linux, it's 127.0.0.1 but i use os x with kubernetes enabled via docker app. which is using 31112.

Reply:
My bad. It shouldn't go there! I'm on fix now

Reply:
Please use 127.0.0.1, why are we seeing port 7777 now?

Reply:
That was bug. Already fixed